ethers-middleware/tests/it/main.rs

```diff
@@ -0,0 +1,49 @@
+#![allow(clippy::extra_unused_type_parameters)]
```

Comment:
all ethers-middleware tests are now on Anvil, and the celo tests moved to tests/live/celo